rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -257,8 +257,6 @@ lv2_socket::lv2_socket(utils::serial& ar, lv2_socket_type _type)
 	ar.pos += 8;
 #endif
 
-	const s32 version = GET_SERIALIZATION_VERSION(lv2_net);
```

Comment:
Do not remove, mark with [[maybe_unused]] instead.

Reply:
Done.